### PR TITLE
Docker instructions

### PR DESCRIPTION
**Pull request by e6**
At the very least, please update http://portia.readthedocs.org/en/latest/installation.html#docker to reflect the correct docker command to run portia:

docker run -i -t --rm \
-v portia-data:/app/slyd/data:rw \
-p 9001:9001 \
--name portia \
portia

This was merged before to the README.md but the fix was not updated on readthedocs.org.

Thank you

### CODE REVIEW

File: DOCKER-README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    git clone https://github.com/scrapinghub/portia
+
+Then inside the Portia directory, run:
+
+    docker build -t portia .
+
+Create a Docker [volume](https://docs.docker.com/engine/userguide/dockervolumes/) for the Portia project data:
+
+    docker create --name portia-data -v /app/slyd/data:rw portia
```
Comment (psychok7) [inline DOCKER-README.md]: @e6 when i run this i get a `Error response from daemon: Invalid bind mount spec "/app/slyd/data:rw": volumeinvalid: Invalid volume specification: '/app/slyd/data:rw'` with `Docker version 1.10.0, build 590d5108` in Ubuntu 14.04 LTS

File: DOCKER-README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Run portia:
+
+     docker run -i -t --rm \
```
Comment (psychok7) [inline DOCKER-README.md]: @e6 when i try running this (without the volume created) i get a:

```
Traceback (most recent call last):
  File "bin/slyd", line 41, in <module>
    splash.server.main()
  File "/usr/local/lib/python2.7/dist-packages/splash/server.py", line 326, in main
    opts, _ = parse_opts(jupyter, argv)
TypeError: add_args() takes no arguments (2 given)

```

File: DOCKER-README.md (hunk 1 above)
Comment (e6) [inline DOCKER-README.md]: type the following command to create the volume where portia will store the data, then run the docker run command and report back --

docker create --name portia-data -v /app/slyd/data:rw portia

**Comment by e6** [comment]
@psychok7 do you get this error as well above the error you posted? 

WARNING: Error loading config file:<user>/.docker/config.json - stat <user>/.docker/config.json: permission denied

**Comment by psychok7** [comment]
@e6 no, after running `docker build -t portia .` i run `docker create --name portia-data -v /app/slyd/data:rw portia` with the current master and the only error i get is `Error response from daemon: Invalid bind mount spec "/app/slyd/data:rw": volumeinvalid: Invalid volume specification: '/app/slyd/data:rw'`

File: DOCKER-README.md (hunk 1 above)
Comment (psychok7) [inline DOCKER-README.md]: @e6 i got this to work if i dont specify the `rw` i.e `docker create --name portia-data -v /app/slyd/data portia` using the latest docker version

File: DOCKER-README.md (hunk 2 above)
Comment (psychok7) [inline DOCKER-README.md]: @e6 even with the volume successfully created i am still running into this issue. Could it be a master problem?

**Comment by psychok7** [comment]
@e6 it turns out i was using master branch instead of `develop`. with develop i dont get the error but i still have to remove the rw if when i create the docker volume

**Comment by freeslugs** [comment]
@e6 @psychok7 i am getting the same errors as you - stuck on:
```python
Traceback (most recent call last):
  File "bin/slyd", line 41, in <module>
    splash.server.main()
  File "/usr/local/lib/python2.7/dist-packages/splash/server.py", line 326, in main
    opts, _ = parse_opts(jupyter, argv)
TypeError: add_args() takes no arguments (2 given
```

**Comment by psychok7** [comment]
@freeslugs if you use the `develop` branch the errors will stop. 

@e6 from what i gathered https://github.com/scrapinghub/portia/pull/330 was merged to `develop` only and not to `master` so you should probably merge this PR to `develop` and not `master` because `master` is broken

**Comment by freeslugs** [comment]
@psychok7 it worked! btw using docker on mac, i guess the port forwarding didnt work so it's available on `http://192.168.99.100:9001`. thanks!

**Comment by ruairif** [comment]
Superseeded by: https://github.com/scrapinghub/portia/pull/700

**Pull request closed without merging**